tests/unittests/Foundation/ReferenceFoundation/TestNSCharacterSet.mm

```diff
@@ -98,11 +98,10 @@
 TEST(NSCharacterSet, Mutables) {
     auto attachmentCharacterUnichar = static_cast<unichar>(0xFFFC);
```

Comment:
alternatively, do an ifdef on __BIG_ENDIAN__, __LITTLE_ENDIAN__

---
In reply to: [73653571](https://github.com/Microsoft/WinObjC/pull/823#discussion_r73653571) [](ancestors = 73653571)